src/MALA.jl

```diff
@@ -0,0 +1,103 @@
+using ForwardDiff: gradient!
+using DiffResults: GradientResult, value, gradient
+
+struct MALA{D} <: MHSampler
+    proposal::D
+end
+
+
+MALAMH(d) = MALA(RandomWalkProposal(d))
+
+
+struct Transition_w∇{T<:Union{Vector, Real, NamedTuple}, L<:Real, G<:Union{Vector, Real, NamedTuple}}
+    params :: T
+    lp :: L
+    ∇ :: G
```

Comment:
So people can select this easily without unicode characters:

```suggestion
    gradient :: G
```